infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -186,8 +190,10 @@ cp -r $LLVM_SRC/compiler-rt/lib/fuzzer $SRC/libfuzzer
 # Cleanup
 rm -rf $LLVM_SRC
 rm -rf $SRC/chromium_tools
-apt-get remove --purge -y $LLVM_DEP_PACKAGES
-apt-get autoremove -y
+if [ $# -eq 0 ] ; then
+  apt-get remove --purge -y $LLVM_DEP_PACKAGES
```

Comment:
do not remove git in base-builder-rust image

Reply:
Please remove this change.
If it's just a matter of installing git, then install it in the install_rust.sh script. There's tons of other stuff here being installed that should be removed. As is this pretty hacky.